enterprise/startup-script/nginxconfiguressl.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+export STORAGE_ACCESS_ACCOUNT=$1
+export STORAGE_ACCESS_KEY=$2
+export FQDN=$3
+export certprint=$4
+export cacerts=$5
+
+sudo docker run -d --name scalityconnect -p 8000:8000 \
+-p 8005:8005 -e FQDN=$FQDN \
+-e AZURE_STORAGE_ACCESS_KEY=$STORAGE_ACCESS_KEY \
+-e AZURE_STORAGE_ACCOUNT=$STORAGE_ACCESS_ACCOUNT \
+--log-opt max-size=5g --log-opt max-file=10 \
+--restart always scality/connect
+
+# exit on any error
+set -e
+
+until apt-get -y update && apt-get -y install nginx
```

Comment:
we should try some finite number of times. 10? and then fail so the person gets an error on deployment.

Reply:
actually, why don't we have nginx pre-installed on the vm image?